README.md

```diff
@@ -23,27 +23,38 @@ If you'd like to help out, please see [CONTRIBUTING.md](CONTRIBUTING.md).
 * Future release
   - Planned: removal of unused signature verification schemes at link-time.
   - Planned: removal of unused protocol versions at link-time.
-* Next release: 0.21.0 (2023-xx-xx)
+* Next release: 0.21.0 (2023-03-29)
   - Support for connecting to peers named with IP addresses.  This means
     rustls now depends on a fork of webpki - `rustls-webpki` - with a suitably
     extended API.
   - *Breaking change*: `StoresClientSessions` trait renamed to `ClientSessionStore` and
     reworked to allow storage of multiple TLS1.3 tickets and avoid reuse of them.
     This is a privacy improvement, see RFC8446 appendix C.4.
-  - *Breaking change*: `rustls::Error` is no longer `PartialEq`.
+  - *Breaking change*: the `DistinguishedNames` type alias no longer exists; the public
+    API now exports a `DistinguishedName` type, and the
+    `ClientCertVerifier::client_auth_root_subjects()` method now returns a
+    `&[DistinctedName]` instead (with the lifetime constrained to the verifier's).
+  - *Breaking change*: the `ClientCertVerifier` methods `client_auth_mandatory()`
+    and `client_auth_root_subjects()` no longer return an `Option`. You can now
+    use an `Acceptor` to decide whether to accept the connection based on information
+    from the `ClientHello` (like server name).
   - *Breaking change*: rework `rustls::Error` to avoid String usage in
     `PeerMisbehavedError`, `PeerIncompatibleError` and certificate errors.
     Especially note that custom certificate verifiers should move to use the
-    new certificate errors.
+    new certificate errors. `Error` is now `non_exhaustive`, and so are the
+    inner enums used in its variants.
   - *Breaking change*: replace `webpki::Error` appearing in the public API
     in `RootCertStore::add`.
   - The number of tickets sent by a TLS1.3 server is now configurable via
     `ServerConfig::send_tls13_tickets`.  Previously one ticket was sent, now
     the default is four.
+  - *Breaking change*: remove deprecated methods from `Acceptor`.
   - *Breaking change*: `AllowAnyAuthenticatedClient` and `AllowAnyAnonymousOrAuthenticatedClient`
     `new` functions now return `Self`. A `boxed` function was added to both types to easily acquire
     an `Arc<dyn ClientCertVerifier>`.
   - *Breaking change*: `NoClientAuth::new` was renamed to `boxed`.
+  - *Breaking change*: the QUIC API has changed to provide QUIC-specific `ClientConnection` and
+    `ServerConnection` types, instead of using an extension trait.
```

Comment:
Here are a couple more breaking changes I saw comparing [v0.20.8 and main](https://github.com/rustls/rustls/compare/v/0.20.8...rustls:main) and going through some of my drafted dependent project updates:

* the Quic `Secrets` constructor was changed to take a `Side` instead of a `bool` in 1d314aad4439d9055e790ebafc7fa12fa5d4bc06
* `export_keying_material`'s result was changed from `Result<(), Error>` to `Result<T, Error>` where `T: AsMut<[u8]>` in 38fdd952bec1ca9ab7c3b25b60026954115561a9
* `sni_hostname` on `ServerConnection` was renamed to `server_name` in 9b8fa405cda7e278af00e012d62002ee5a6c6a3a
* `DigtallySignedStruct` was moved from `internal::msgs::handshake` to `rustls` in f14e709209b5996312b5b4c34c9fdbb20a37e692 (_maybe not necessary to list this one since `internal` wasn't part of the supported API?_)

Reply:
Ah nice, do you want to force push to this branch to add these?

Reply:
Sure, can do.

Reply:
Done :ballot_box_with_check: 

Here's the [diff](https://github.com/rustls/rustls/compare/b3d71df9311d756b7567a33e5bafc82bea4258a1..f5039da583dea9b4175820ef634afa65263ca9e8) of what I added.